### PR TITLE
refactor(matchbox-plugin):changing api upperCase fields to lowerCase

### PR DESCRIPTION
**Pull request by AshotManukyan**
## Description

[Monday ticket](https://applicaster.monday.com/boards/1615228456/pulses/5106153946)

Changed all the fields that started with a capital letter to be lowerCase

<details>
  <summary>Platforms Affected</summary>
  
- [x] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [x] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/quick-brick-matchbox/README.md [hunk 1]
```diff
@@ -93,17 +91,15 @@ Tested on Android mobile and ios Mobile.
 Inside each entry in ``extensions``: 
     1. `team1_name` / `team2_name`;
     2. `team1_pic` / `team2_pic`;
-    3. `TimeOfGame`;
-    4. `PlaceOfGameHeb`;
+    3. `timeOfGame`;
+    4. `placeOfGameHeb`;
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/README.md:95]: should be placeOfGame without Heb

File: plugins/quick-brick-matchbox/src/components/MatchBoxItem.js [hunk 2]
```diff
@@ -1,271 +1,271 @@
...
+} = extensions;
 
-  const matchStatusTitle = getMatchStatusTitle(matchStatus, localization);
-  const styles = getStyles(customStyles);
+const matchStatusTitle = getMatchStatusTitle(matchStatus, localization);
+const styles = getStyles(customStyles);
 
-  const buttonHandler = (params) => {
+//this is for api changed fields
+const placeOfGameHeb = extensions?.PlaceOfGameHeb || extensions?.placeOfGameHeb;
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/src/components/MatchBoxItem.js:28]: remove the Heb

File: plugins/quick-brick-matchbox/src/utils/consts.js [hunk 3]
```diff
@@ -4,6 +4,8 @@ export const MATCH_STATUS_FUTURE = 2;
 
 export const MATCH_STATUS_LIVE = 3;
 
+export const BOARD_ID = 5;
```
Comment (snooky23) [inline plugins/quick-brick-matchbox/src/utils/consts.js:7]: What the number represent?

**Review by snooky23 — CHANGES_REQUESTED** [review]

File: plugins/quick-brick-matchbox/src/utils/consts.js (hunk 3 above)
Comment (AshotManukyan) [reply plugins/quick-brick-matchbox/src/utils/consts.js:7]: This was hardcoded in MatchboxItem component like this
`{Number(extensions?.BOARD_ID) === 5 && (
              <Text
                style={styles.gameTitle}
              >{`${localization?.gameNumber} ${extensions?.gameNumber}`}</Text>
            )}`
so I removed from render and placed it in constants

**Review by snooky23 — APPROVED** [review]

**Pull request merged**